tests/data/app.yaml

```diff
@@ -0,0 +1,10 @@
+# Dummy app.yaml to placate nosegae.
```

Comment:
It parses the app.yaml to make sure the right versions of any declared libraries are used, I think. (at least, that's what ferrisnose did)